src/test/java/core/basesyntax/service/impl/FileReaderCsvTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.PurchaseHandler;
+import core.basesyntax.strategy.impl.ReturnHandler;
+import core.basesyntax.strategy.impl.SupplyHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
```

Comment:
To enhance readability and conciseness, consider using static imports for assertions used in the test classes.